velox/common/memory/tests/SharedArbitratorTest.cpp

```diff
@@ -2747,6 +2747,61 @@ DEBUG_ONLY_TEST_F(SharedArbitrationTest, arbitrateMemoryFromOtherOperator) {
   }
 }
 
+TEST_F(SharedArbitrationTest, joinBuildSpillError) {
```

Comment:
Would it make sense to run this test in "stress" mode to make sure it is not flaky?

Reply:
Sure.